source/_components/notify.html5.markdown

```diff
@@ -33,30 +33,32 @@ Configuration variables:
 
 ### {% linkable_title Getting ready for Chrome %}
 
-1. Create new project at [https://console.cloud.google.com/home/dashboard](https://console.cloud.google.com/home/dashboard).
-2. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain.
-3. After that, go to [https://console.firebase.google.com](https://console.firebase.google.com) and select import Google project, select the project you created.
-4. Then, click the cogwheel on top left and select "Project settings".
-5. Select 'Cloud Messaging' tab, listed beneath Project Credentials will be your 152 character 'Server Key' and 12 digit ID 'Sender ID'.
-
-#### {% linkable_title Verify your domain with Hass.io %}
-
-1. Set `https://yourdomain/local/` as URL.
-2. For verifying your domain you need to download a file in step 2.
-3. Create a directory named "www" in you Hass.io configuration directory.
-4. Place the file (something like this: google*.html) in the "www" directory.
-5. Verify the domain.
-6. Proceed with step 3.
+1. Make sure you can access your Home Assistant installation from outside your network over https ([see docs](https://home-assistant.io/docs/configuration/remote/)).
+2. Create a new project at [https://console.cloud.google.com/home/dashboard](https://console.cloud.google.com/home/dashboard).
+3. Go to [https://console.cloud.google.com/apis/credentials/domainverification](https://console.cloud.google.com/apis/credentials/domainverification) and verify your domain via Google Webmaster Central / Search Console - [instructions](#verify-your-domain).
+4. With the domain verified, go to [https://console.firebase.google.com](https://console.firebase.google.com), select import Google project, and select the project you created.
+5. Then, click the cogwheel on top left and select "Project settings".
+6. Select 'Cloud Messaging' tab, listed beneath Project Credentials will be your 152 character 'Server Key' and 12 digit ID 'Sender ID' you need for configuring this component.
+
+#### {% linkable_title Verify your domain %}
+
+Follow these steps to verify domain ownership with Google Webmaster Central / Search Console:
+1. Enter your domain and add **'/local'** at the end, ie. https://whitehouse.duckdns.org:8123/local
```

Comment:
Only one thing I think now - `whitehouse.duckdns.org` is a real domain, use `duckdns.example.org`?